.github/workflows/release-docs.yml

```diff
@@ -0,0 +1,22 @@
+name: Publish docs [release]
+
+on:
+  release:
+    types: [published]
+
+permissions:
+  contents: write
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.10.6
```

Comment:
Note that Netlify was previously pinned to 3.8 for builds.

I've just updated the `PYTHON_VERSION` environment variable in Netlify to `3.10.6`, so that deployment previews on PRs match the release deployments.